pkg/conversion/capi2mapi/common.go

```diff
@@ -124,8 +133,8 @@ func convertCAPIMachineAnnotationsToMAPIMachineSpecObjectMetaAnnotations(capiAnn
 	return mapiAnnotations
 }
 
-func convertCAPIAnnotationsToMAPIAnnotations(capiAnnotations map[string]string) map[string]string {
-	if len(capiAnnotations) == 0 {
+func convertCAPIAnnotationsToMAPIAnnotations(capiAnnotations map[string]string, machineAPIAnnotations map[string]string) map[string]string {
```

Comment:
We may want to change the param name based on what we decide on https://github.com/openshift/cluster-capi-operator/pull/393/files#r2487307586